src/global/exception/common/global-exception.filter.ts

```
@@ -0,0 +1,76 @@
+            let message: string | string[] = '알 수 없는 오류가 발생했습니다.';
+
+            if (exception instanceof HttpException) {
+                const exceptionResponse = exception.getResponse();
+                // 메시지가 문자열 또는 객체인 경우 처리
+                if (typeof exceptionResponse === 'string') {
+                    message = exceptionResponse;
+                } else if (
+                    typeof exceptionResponse === 'object' &&
+                    exceptionResponse.hasOwnProperty('message')
+                ) {
+                    message = (exceptionResponse as any).message;
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)